var/spack/repos/builtin/packages/m4/package.py

```diff
@@ -19,6 +19,8 @@ class M4(AutotoolsPackage, GNUMirrorPackage):
     patch('gnulib-pgi.patch', when='@1.4.18')
     patch('pgi.patch', when='@1.4.17')
     patch('nvhpc.patch', when='@1.4.18 %nvhpc')
+    patch('nvhpc-1.4.19.patch', when='@1.4.19 %nvhpc')
+    patch('nvhpc-long-width.patch', when='@1.4.19 %nvhpc')
```

Comment:
Can you add a comment above each of these two patches with information on the provenance and what they do?

Reply:
Added